azurerm/internal/services/blueprints/blueprint_assignment_resource.go

```diff
@@ -311,7 +311,9 @@ func resourceArmBlueprintAssignmentDelete(d *schema.ResourceData, meta interface
 	name := assignmentID.Name
 	targetScope := assignmentID.Scope
 
-	resp, err := client.Delete(ctx, targetScope, name)
+	// We use none here to align the previous behaviour of the blueprint resource
+	// TODO: we could add a features flag for the blueprint to empower terraform when deleting the blueprint to delete all the generated resources as well
```

Comment:
I think we should remove this as it may or may not actually be a good idea according to @jackofallops 
```suggestion
```